addons/web/static/src/js/views/list/list_editable_renderer.js

```diff
@@ -659,7 +659,7 @@ ListRenderer.include({
         if (!thElements.length) {
             return;
         }
-        const table = this.el.getElementsByTagName('table')[0];
+        const table = this.el.getElementsByClassName('o_list_table')[0];
```

Comment:
@aab-odoo, @ged-odoo Hello, could you take a look at this commit ? We would like to add a custom dashboard on top of the purchase list view like so : 
![image](https://user-images.githubusercontent.com/12071695/77925918-c0522280-72a5-11ea-93ca-132590f317b7.png)
Thanks a lot